### DISCUSSION TITLE
Native accelerometer & gyro support - telem & gyro stabilizer

### DISCUSSION THREAD

**Discussion by vidma** [ExpressLRS/ExpressLRS#2353] [Ideas]
Maybe dumb idea, but as there's already a native baro support  https://github.com/ExpressLRS/ExpressLRS/pull/1892 , another one would be to add native support for gyro/accelerometer directly connected directly to ELRS RX chip:
- accelerometer / g-sensor (some code+driver seem to exist for some different usecase https://github.com/ExpressLRS/ExpressLRS/tree/567bc8c4fbb574330af70f9162586a43e2bbaaca/src/lib/GSENSOR , for powersave? 
- gyro (not existent it seems)

this could be used for saving $ skipping FC to either:
- just report extra telemetry, like angle of attack & acceleration
- more fun: implement a **dumb gyro (mini-FC) using ideally a PID-loop based stabilization (or something simpler) for wings**, so no external gyro would be needed (no point in reinventing smart thing like Ardupilot, and such code wouldn't fit...). so this would be a cheaper alternative to FC, e.g. **for kids at school (10$ for ELRS +10$ for sensor, vs 50+$ for FC)**, or when simple gyro is enough or whatever
- even dumber idea: how about run Ardupilot code remotely :D on TX or PC (SITL?), but i guess telemetry delay might screw things up a bit :D 

what do you think?

**Comment by pkendall64** [comment]
There is already a project underway for this https://discord.com/channels/596350022191415318/1135438746016292914

**Reply by vidma** [reply]
cool!
 ~is it a private discord channel/thread? Unable to join the thread~
found that i need first join https://discord.com/invite/expresslrs
